build/xunit-core.runsettings

```diff
@@ -1,5 +1,5 @@
 <RunSettings>
     <RunConfiguration>
-        <ReporterSwitch>verbose</ReporterSwitch>
```

Comment:
Removing `ReporterSwitch` means that test logs no longer report starting/stopping messages, so when tests hang (unfortunately still common on Linux) we have no ability to detect which ones started and never finished.

Reply:
I removed ReporterSwitch beause it was not recognized by vstest. There's no docs for that option in the link I posted in the [description](https://github.com/MicrosoftDocs/visualstudio-docs/blob/master/docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md#runconfiguration-element). The verbosity is specified in tthe command line

Reply:
runsettings file:

```
<RunSettings>
    <RunConfiguration>
        <ReporterSwitch>verbose</ReporterSwitch>
        <TargetFrameworkVersion>FrameworkCore10</TargetFrameworkVersion>
    </RunConfiguration>
</RunSettings>
```

Sample command run:

```
C:\src\NuGet.Client\cli\dotnet.exe ^
vstest ^
--Settings:C:\src\NuGet.Client\build\xunit-core.runsettings ^
--Diag:diag.log;tracelevel=verbose ^
C:\src\NuGet.Client\test\NuGet.Core.Tests\NuGet.Protocol.Tests\bin\Debug\netcoreapp5.0\win7-x64\NuGet.Protocol.Tests.dll
```

Output in diag.log

```
TpTrace Warning: 0 : 5148, 1, 2020/08/24, 10:15:46.226, 346912292171, vstest.console.dll, Invalid settings 'RunConfiguration'. Unexpected XmlElement: 'ReporterSwitch'.
TpTrace Warning: 0 : 5148, 1, 2020/08/24, 10:15:46.226, 346912294802, vstest.console.dll, Invalid settings 'RunConfiguration'. Unexpected XmlElement: 'ReporterSwitch'.
```

Reply:
It's an xunit specific setting, so I'm not surprised that mstest doesn't recognise it. You can check CI logs and compare the output.

Reply:
but xunit respects it right?

https://xunit.net/releases/2.4-beta1

Reply:
CI logs in Build and E2E logs does not show any verbose logs for test runs. I will revert the option.

Reply:
@nkolev92 I couldn't make it work with my sample run. Let's see what happens on the CI.

```
C:\src\NuGet.Client>C:\src\NuGet.Client\cli\dotnet.exe vstest --Settings:C:\src\NuGet.Client\build\xunit-core.runsettings --Diag:diag.log;tracelevel=verbose --logger:"xunit;LogFileName=test-xunit.xml" --logger:"console;verbosity=detailed" C:\src\NuGet.Client\test\NuGet.Core.Tests\NuGet.Protocol.Tests\bin\Debug\netcoreapp5.0\win7-x64\NuGet.Protocol.Tests.dll
Microsoft (R) Test Execution Command Line Tool Version 16.8.0-preview-20200811-01
Copyright (c) Microsoft Corporation.  All rights reserved.

Starting test execution, please wait...
Logging Vstest Diagnostics in file: C:\src\NuGet.Client\diag.log
Test run detected DLL(s) which were built for different framework and platform versions. Following DLL(s) do not match current settings, which are .NETCoreApp,Version=v1.0 framework and X64 platform.
NuGet.Protocol.Tests.dll is built for Framework net5.0 and Platform AnyCPU.
Go to http://go.microsoft.com/fwlink/?LinkID=236877&clcid=0x409 for more details on managing these settings.
Could not find a test logger with AssemblyQualifiedName, URI or FriendlyName 'xunit'.
```

Reply:
Seems to work on the CI: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=4006064&view=logs&j=b73baef3-01cd-534b-3785-53f5a860ca9d&t=3e715b0a-c3d8-57c0-fcd0-3bed531e3dfa